go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -227,6 +235,128 @@ func (c *Controller) handleErr(err error, key interface{}) {
 	utilruntime.HandleError(err)
 }
 
+// initTopLevelCache will take load balancer data currently applied in OVN and populate the cache.
+// An important caveat here is that no effort is made towards populating some details of ovnlb.LB here.
+// That is because such work will be performed in syncService, so all that is needed here is the ability
+// to distinguish what is present in ovn database and this 'dirty' initial value.
+func (c *Controller) initTopLevelCache() error {
+	c.alreadyAppliedLock.Lock()
+	defer c.alreadyAppliedLock.Unlock()
+
+	// first, list all load balancers and their respective services
+	services, lbs, err := getServiceLBs(c.nbClient)
+	if err != nil {
+		return fmt.Errorf("failed to load balancers: %w", err)
+	}
+
+	c.alreadyApplied = make(map[string][]ovnlb.LB, len(services))
+
+	for _, lb := range lbs {
+		service := lb.ExternalIDs["k8s.ovn.org/owner"]
```

Comment:
Maybe it's a good time to create some constant for `k8s.ovn.org/owner`?

Reply:
done